### ISSUE TITLE
Empty `config-get` throws an error

### ISSUE THREAD

**Issue by DaniFoldi** [ddworken/hishtory#27]
Hi :wave:,

I'm working on an alternative hishtory backend that could be hosted on Cloudflare Workers with D1 (cool serverless stuff). I was looking for the place where some base URL for the backend is stored (for future reference, it is the `HISHTORY_SERVER` env variable - a mention of this in the docs would be appreciated), so I tried an empty `hishtory get-config` with no key, which resulted in an index out of range error, as seen on the screenshot below. _After some digging in the code I found the command handling switch and found that the URL is not actually part of the configuration._

Maybe instead of the error it could list the available config keys?

![Screenshot 2022-11-13 at 15 52 01](https://user-images.githubusercontent.com/6977583/201530948-0c96ea56-5dd4-4c3d-9112-9bf93acd1dea.png)

**Comment by ddworken** [comment]
Ooo, I love the idea of porting this to Cloudflare Workers + D1, I've thought about doing the exact same thing. :) If you end up getting it to work, please do share and I'd be happy to link it in the README! And let me know if you have any questions about how anything works. 

But +1 to this FR. Modernizing the CLI handling for hiSHtory is on my todo list and I'll make sure to tackle this then.

**Comment by ddworken** [comment]
Fixed in `v0.181` which ported all CLI handling to use cobra-cli  🎉 

```
david@Davids-MacBook-Air hishtory % hishtory config-get
Get the value of a config option

Usage:
  hiSHtory config-get [flags]
  hiSHtory config-get [command]

Available Commands:
  custom-columns            The list of custom columns that hishtory is tracking
  displayed-columns         The list of columns that hishtory displays
  enable-control-r          Whether hishtory replaces your shell's default control-r
  filter-duplicate-commands Whether hishtory filters out duplicate commands when displaying your history
  timestamp-format          The go format string to use for formatting the timestamp

Flags:
  -h, --help   help for config-get

Use "hiSHtory config-get [command] --help" for more information about a command.
```
